container/Dockerfile

```diff
@@ -1,10 +1,21 @@
 FROM registry.access.redhat.com/ubi7/ubi:latest
 LABEL maintainer "Chen Gao <Chen.Gao@.ibm.com>"
 
-ENV IOT_REPO_PATH=/root/db2eventstore-IoT-Analytics \
-    KAFKA_REPO_PATH=/root/db2eventstore-kafka
-ARG SETUP_AREA=${IOT_REPO_PATH}/container/setup
+ENV IOT_REPO_PATH=/root/db2eventstore-IoT-Analytics
+ENV KAFKA_REPO_PATH=/root/db2eventstore-kafka 
+ENV SPARK_HOME="/spark_home"
+ENV PATH=$PATH:$SPARK_HOME/bin:$SPARK_HOME/sbin
+ARG ES_VERSION=latest
 ARG BRANCH=master
+ARG SETUP_AREA=${IOT_REPO_PATH}/container/setup
+
+
+#breakpoints
```

Comment:
I'm sure these are just temporary for debugging but just a reminder to remove these prior to the delivery.

Reply:
done